Userland/Libraries/LibJS/Bytecode/ASTCodegen.cpp

```diff
@@ -932,6 +932,11 @@ void YieldExpression::generate_bytecode(Bytecode::Generator& generator) const
     generator.switch_to_basic_block(continuation_block);
 }
 
+void AwaitExpression::generate_bytecode(Bytecode::Generator&) const
+{
+    TODO();
+}
+
```

Comment:
I believe the default `generate_bytecode` also has a TODO with a message saying what it is so that could be more clear.

Reply:
You're right, fixed